src/main/java/io/supertokens/storage/mysql/queries/JWTSigningQueries.java

```diff
@@ -0,0 +1,99 @@
+/*
+ *    Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ *    This software is licensed under the Apache License, Version 2.0 (the
+ *    "License") as published by the Apache Software Foundation.
+ *
+ *    You may not use this file except in compliance with the License. You may
+ *    obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *    License for the specific language governing permissions and limitations
+ *    under the License.
+ */
+
+package io.supertokens.storage.mysql.queries;
+
+import io.supertokens.pluginInterface.RowMapper;
+import io.supertokens.pluginInterface.exceptions.StorageQueryException;
+import io.supertokens.pluginInterface.jwt.JWTAsymmetricSigningKeyInfo;
+import io.supertokens.pluginInterface.jwt.JWTSigningKeyInfo;
+import io.supertokens.pluginInterface.jwt.JWTSymmetricSigningKeyInfo;
+import io.supertokens.storage.mysql.Start;
+import io.supertokens.storage.mysql.config.Config;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class JWTSigningQueries {
+    static String getQueryToCreateJWTSigningTable(Start start) {
+        return "CREATE TABLE IF NOT EXISTS " + Config.getConfig(start).getJWTSigningKeysTable() + " ("
+                + "key_id VARCHAR(255) NOT NULL," + "key_string TEXT NOT NULL,"
+                + "algorithm VARCHAR(10) NOT NULL," + "created_at BIGINT UNSIGNED,"
+                + "PRIMARY KEY(key_id));";
+    }
+
+    public static List<JWTSigningKeyInfo> getJWTSigningKeys_Transaction(Start start, Connection con)
+            throws SQLException, StorageQueryException {
+        String QUERY = "SELECT * FROM "
+                + Config.getConfig(start).getJWTSigningKeysTable()
+                + " ORDER BY created_at DESC FOR UPDATE;";
```

Comment:
Check if you need to add a ";" in the query.

Reply:
It works without it, removed the semicolon